packages/storybook/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "storybook",
+  "version": "0.0.0",
+  "description": "Storybook build for core package",
+  "private": true,
+  "scripts": {
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook --output-dir dist"
```

Comment:
Not just `build` and `start`?

Reply:
Switched to `start` but I don't want build in the repo root to trigger the storybook build, and think this is better than manually excluding n shite. Only thing that is really gonna be building the storybook is the CI action anyway.